### PR TITLE
Azure marketplace

### PR DESCRIPTION
**Pull request by kelseiv**
Updates for Enterprise listing on Azure Marketplace

### CODE REVIEW

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/_index.md [hunk 1]
```diff
@@ -20,8 +20,12 @@ To deploy an InfluxDB Enterprise cluster or a multi-cluster architecture on Amaz
 
 InfluxDB Enterprise is currently available on the Google Cloud Platform (GCP). For information on deploying InfluxDB Enterprise clusters on Google Cloud Platform (GCP), see [Deploy an InfluxDB Enterprise cluster on Google Cloud Platform](/enterprise_influxdb/v1.8/install-and-deploy/deploying/google-cloud-platform/).
 
+## [Deploy on Azure Cloud Platform](/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure/)
+
+InfluxDB Enterprise is currently available on the Azure Cloud Platform (GCP). For information on deploying InfluxDB Enterprise clusters on Azure, see [Deploy an InfluxDB Enterprise cluster on Azure Cloud Platform](/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure/).
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/_index.md:25]: > the Azure Cloud Platform (GCP)
Remove the `(GCP)`

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Prerequisites
+
+This guide requires the following:
+
+- Microsoft Azure account with access to the [Azure Marketplace](https://cloud.google.com/marketplace/).
+- A valid InfluxDB Enterprise license key, or [sign up for a free InfluxDB Enterprise trial](?).
+- Access to [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701), cmd.exe, or [PowerShell](https://github.com/PowerShell/PowerShell).
+
+To deploy InfluxDB Enterprise on platforms other than Azure, see [Deploy InfluxDB Enterprise clusters in containers](/enterprise_influxdb/v1.8/install-and-deploy/_index).
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:22]: Remove "in containers"

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Deploy a cluster
+
+To deploy an InfluxDB Enterprise cluster, log in to your Google Cloud Platform account and navigate to [InfluxData's InfluxDB Enterprise (BYOL)](https://console.cloud.google.com/partner/editor/influxdata-public/influxdb-enterprise-byol) solution in the GCP Marketplace.
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:26]: Update to Azure.

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Click __Launch on compute engine__ to open up the configuration page.
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:28]: These steps will need to be updated once you get access to Azure.

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+The cluster will take up to five minutes to fully deploy. If the deployment does not complete or reports an error, read through the list of [common deployment errors](https://cloud.google.com/marketplace/docs/troubleshooting).
+
+Your cluster is now deployed!
+
+> **Note:** Make sure you save the "Admin username", "Admin password", and "Connection internal IP" values displayed on the screen. They will be required when attempting to access the cluster.
+
+## Access the cluster
+
+The cluster's IP address is only reachable from within the GCP network (or subnetwork) specified in the solution configuration. A cluster can only be reached from instances or services within the same GCP network or subnetwork in which it was provisioned.
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:44]: Access will be similar to the GCP cluster, except the Chronograf instance will be an access portal if the user decides to create one.

Comment (kelseiv) [reply content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:44]: @gunnaraasen do we know commands to create a new instance to access InfluxDB Enterprise cluster (and Chronograf)?
Also, what's command for SSHing into the instance?

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 6]
```diff
@@ -7,43 +7,47 @@ menu:
...
+3. Select the subscription to use for your InfluxDB Enterprise cluster, and then select the resource group to access the cluster.
 
-Adjust any other fields as desired. The cluster will only be accessible within the network (or subnetwork, if specified) in which it is deployed. The fields in collapsed sections generally do not need to be altered.
+   **Tip:** We recommend creating a new resource group to use for your InfluxDB Enterprise cluster.
 
-Click __Deploy__ to launch the InfluxDB Enterprise cluster.
+4. Select instance details, including the cluster region, user name of the cluster administrator, authentication type. For password authentication, enter your user name and password. For SSH public key authentication, enter your user name and SSH key.
 
-The cluster will take up to five minutes to fully deploy. If the deployment does not complete or reports an error, read through the list of [common deployment errors](https://cloud.google.com/marketplace/docs/troubleshooting).
+5. Click **Next: Cluster Configuration**, and then enter details including the database admin username and password, the number of meta and data nodes, and the VM size for both meta and data nodes.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:35]: any guidelines for VM sizing?

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 7]
```diff
@@ -7,43 +7,47 @@ menu:
...
 
-Your cluster is now deployed!
+6. Click **Next: External Access & Chronograf**, and then do the following:
 
-> **Note:** Make sure you save the "Admin username", "Admin password", and "Connection internal IP" values displayed on the screen. They will be required when attempting to access the cluster.
+   - To install Chronograf as a separate node, select **Yes**. Otherwise, select **No**.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:39]: Is No the default option?

File: content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md [hunk 8]
```diff
@@ -7,43 +7,47 @@ menu:
...
+2. On the InfluxDB Enterprise (Official Version) page, click **Create**.
 
-Copy the InfluxDB Enterprise license key to the __InfluxDB Enterprise license key__ field or [sign up for a free InfluxDB Enterprise trial for GCP](https://portal.influxdata.com/users/gcp) to obtain a license key.
+3. Select the subscription to use for your InfluxDB Enterprise cluster, and then select the resource group to access the cluster.
 
-Adjust any other fields as desired. The cluster will only be accessible within the network (or subnetwork, if specified) in which it is deployed. The fields in collapsed sections generally do not need to be altered.
+   **Tip:** We recommend creating a new resource group to use for your InfluxDB Enterprise cluster.
 
-Click __Deploy__ to launch the InfluxDB Enterprise cluster.
+4. Select instance details, including the cluster region, user name of the cluster administrator, authentication type. For password authentication, enter your user name and password. For SSH public key authentication, enter your user name and SSH key.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/install-and-deploy/deploying/azure.md:33]: Does it make sense to recommend using the same username for the cluster admin and db admin?

**Comment by kelseiv** [comment]
> I've pushed some updates. Let me know if this looks good now!

Looks, great! Thank you, Gunnar!

**Pull request merged**